File: practice codes/Ifelse.java
import java.util.*;
public class Ifelse
{
    public static void main(String[] args)
     {

        Scanner sobj = new Scanner(System.in);

        int no = 0;
        System.out.println("Enter first number :");
        no = sobj.nextInt();

        if (no>=18)
        {
            System.out.println("You are eligible to Vote");
            
        }
        else
        {
            System.out.println("You are not eligible to vote");
        }
     }
}

File: practice codes/Ifelse.c
#include<stdio.h>
int main()
{
    int no1 , no2, result  ;

    printf("Enter first number :");
    scanf("%d",&no1);

    printf("Enter Second number :");
    scanf("%d",&no2);

    result = no1 + no2;

    if (result >= 120)
    {
        printf("Congratulatios you are selected");
    }
    else 
    {
        printf("Better luck next time !!");
    }
    


    return 0;
}